upgrade_test.py

```diff
@@ -813,10 +813,14 @@ def _add_sla_credentials_to_stress_commands(self, workloads_with_sla: list):
 
         roles = []
         service_level_shares = self.params.get("service_level_shares")
+        # OSS Scylla version does not support "shares" option for Service Level
+        if service_level_shares and not self.db_cluster.nodes[0].is_enterprise:
+            service_level_shares = [None for _ in service_level_shares]
+
         with self.db_cluster.cql_connection_patient(node=self.db_cluster.nodes[0], user=loader_utils.DEFAULT_USER,
                                                     password=loader_utils.DEFAULT_USER_PASSWORD) as session:
             for index, shares in enumerate(service_level_shares):
-                roles.append(create_sla_auth(session=session, shares=shares, index=index))
+                roles.append(create_sla_auth(session=session, shares=shares, index=str(index)))
```

Comment:
@juliayakovlev 

now that I'm backporting it `create_sla_auth` signature says index is int